Classes/ios/Scanners/MTBBarcodeScanner.m

```diff
@@ -716,17 +722,18 @@ - (BOOL)isCapturingStillImage {
 
 #pragma mark - Setters
 
-- (void)setCamera:(MTBCamera)camera {
```

Comment:
And, as part of this change, the README will need to be updated, which currently says the below code will change the camera:

```
MTBBarcodeScanner *scanner = [[MTBBarcodeScanner alloc] initWithPreviewView:_previewView];
scanner.camera = MTBCameraFront;
```

But, this will break with the latest change, so let's move people over to using:

```
NSError *error = nil;
MTBBarcodeScanner *scanner = [[MTBBarcodeScanner alloc] initWithPreviewView:_previewView];
[scanner setCamera:MTBCameraFront error:&error]
```

And, as part of this, we can make the `setCamera:error:` method public.